### DISCUSSION TITLE
Feature Request: Allow reordering within category field

### DISCUSSION THREAD

**Discussion by tigerchick** [craftcms/cms#8748] [Ideas] [answered]
In some circumstances it would be useful to be able to reorder selected items within a category field - in the same way that this is possible in an Entries field.

There is a mention of it here: https://craftcms.stackexchange.com/questions/19308/custom-category-order-in-a-categories-field but it is currently not possible.

**Comment by brandonkelly** [comment, marked as answer]
We’ve made this possible for Craft 4.4 🎉 (#11088)

Categories fields now have a “Maintain hierarchy” setting, which can be disabled to cause the fields to behave like Entries fields. (Note the fields will no longer automatically relate ancestors, as well.)

**Reply by brandonkelly** [reply]
Craft 4.4 is out with that change!
